apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/user-bookings.e2e-spec.ts

```diff
@@ -2553,6 +2553,111 @@ describe("Bookings Endpoints 2024-08-13", () => {
       });
     });
 
+    describe("cant't cancel already cancelled booking", () => {
+      it("should not be able to cancel original rescheduled booking", async () => {
+        const originalRescheduledBooking = await bookingsRepositoryFixture.create({
+          status: "CANCELLED",
+          rescheduled: true,
+          user: {
+            connect: {
+              id: user.id,
```

Comment:
Between the two tests there is just a difference of rescheduled:true and the expected error message.

Either we could have functions to createCancelledBooking and createRescheduledBooking to reuse stuff or we could just keep the code itself simple and not going into details whether it was rescheduled, avoiding the second test itself.